ui/components/multichain/notification-list-item-text/notification-list-item-text.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { FC } from 'react';
+
+import { Text } from '../../component-library';
+import {
+  FontWeight,
+  TextColor,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { getRandomKey } from '../../../helpers/utils/notification.util';
+
+export interface NotificationListItemTextItemProps {
+  text: string;
+  highlighted?: boolean;
+}
+
+export interface NotificationListItemTextProps {
+  items: NotificationListItemTextItemProps[];
+  variant?: TextVariant;
+  color?: TextColor;
+}
+
+/**
+ * A component to render a list of notification item titles
+ *
+ * @param props - The properties of the component
+ * @param props.items - The list of items to render
+ * @param props.variant - The text variant of the component
+ * @param props.color - The text color of the component
+ * @returns A JSX element
+ */
+export const NotificationListItemText: FC<NotificationListItemTextProps> = ({
+  items,
+  variant = TextVariant.bodySm,
+  color = TextColor.textDefault,
+}) => {
+  const renderItems = items.map(({ text, highlighted }, index) => {
+    const key = getRandomKey(text, index);
```

Comment:
Minor note. I don't think we need to "generate" a unique key. Using an index here for the key is fine.

This component won't be re-rendering with its list items in a different order, so should be fine to stick with an index.

This is minor, so you can perform cleanup later.